jina/serve/runtimes/worker/http_sagemaker_app.py

```diff
@@ -145,6 +154,47 @@ async def post(request: Request):
                         detail='Invalid CSV input. Please check your input.',
                     )
 
+                def construct_model_from_line(
+                    model: Type[BaseModel], line: List[str]
+                ) -> BaseModel:
+                    parsed_fields = {}
+                    model_fields = model.__fields__
+
+                    for field_name, field_str, field_info in zip(
+                        model_fields.keys(), line, model_fields.values()
```

Comment:
this looks like a very weird way to iterate.

```suggestion
                        for field_str, (field_name, field_info) in zip (line, model_fields.items())
```

Reply:
ah right